src/io/functions/iologindata_save_player.cpp

```diff
@@ -327,16 +327,19 @@ bool IOLoginDataSave::savePlayerStash(std::shared_ptr<Player> player) {
 		return false;
 	}
 
+	query.str("");
+
+	DBInsert stashQuery("INSERT INTO `player_stash` (`player_id`,`item_id`,`item_count`) VALUES ");
 	for (const auto &[itemId, itemCount] : player->getStashItems()) {
-		query.str("");
-		query << "INSERT INTO `player_stash` (`player_id`,`item_id`,`item_count`) VALUES (";
-		query << player->getGUID() << ", ";
-		query << itemId << ", ";
-		query << itemCount << ")";
-		if (!db.executeQuery(query.str())) {
+		query << player->getGUID() << ',' << itemId << ',' << itemCount;
+		if (!stashQuery.addRow(query)) {
 			return false;
 		}
 	}
+
+	if (!stashQuery.execute()) {
```

Comment:
o return stashQuery.execute() retorna o valor direto da função, 
se for true retorna true e se for false retorna false,stashQuery.execute vai retornar false se não vai retornar true. Acho que não precisaria

Reply:
Eu mantive essa redundância pois permanece igual as restantes das funções saveX. Provável que a ideia seja ajudar no debug.